browser/sources.gni

```diff
@@ -356,7 +356,7 @@ if (is_mac) {
   brave_chrome_framework_bundle_deps = [ "//brave:framework_bundle_data" ]
   brave_chrome_framework_contents = [ "Frameworks" ]
   brave_chrome_framework_ldflags = []
-  if (enable_stripping) {
+  if (!is_component_build) {
```

Comment:
this should be `!is_component_build && chrome_pgo_phase != 1` to match up with the guards on `verify_chrome_framework_order`